packages/gatsby-source-vtex/src/gatsby-node.ts

```diff
@@ -51,74 +67,97 @@ export interface Options extends PluginOptions, VTEXOptions {
 }
 
 const DEFAULT_PAGE_TYPES_WHITELIST = [
-  'Product',
   'Department',
   'Category',
   'Brand',
   'SubCategory',
 ]
 
+const readFile = promisify(readFileAsync)
```

Comment:
nice suggestion. Then I'd need to import from `fs/promises` instead of just 'fs'

Reply:
Hmm, good point. I think we can increase our node to v14, because we are in alpha, and after alpha will be more difficult to change. I'm only seeing good points bumping the node dependency, what do you think about it?